Creating release binaries?

I'm trying to cross-compile a fairly recent version of master (*actually [this branch](https://github.com/prologic/nomad/tree/update_go-getter) which I have #5446 up for*) using the `prerelease` and `release` targets in your [GNUMakefile](/GNUMakefile) as per documentation.

This doesn't work however:

```#!bash
james@Jamess-MacBook-Pro
Wed Mar 20 15:27:27
~/go/src/github.com/hashicorp/nomad
 (update_go-getter) 2
$ make prerelease && make release
==> Linting source code...
client/allocrunner/taskrunner/task_runner_test.go:713:27:warning: unnecessary conversion (unconvert)
command/agent/config_test.go:987:35:warning: unnecessary conversion (unconvert)
command/agent/config_test.go:946:35:warning: unnecessary conversion (unconvert)
command/agent/config_test.go:926:35:warning: unnecessary conversion (unconvert)
command/agent/config_test.go:905:35:warning: unnecessary conversion (unconvert)
command/agent/config_test.go:965:35:warning: unnecessary conversion (unconvert)
drivers/shared/executor/executor_test.go:230:35:warning: unnecessary conversion (unconvert)
drivers/shared/executor/executor_test.go:144:36:warning: unnecessary conversion (unconvert)
nomad/autopilot_test.go:222:29:warning: unnecessary conversion (unconvert)
nomad/stats_fetcher_test.go:71:34:warning: unnecessary conversion (unconvert)
nomad/stats_fetcher_test.go:72:48:warning: unnecessary conversion (unconvert)
scheduler/generic_sched_test.go:625:22:warning: unnecessary conversion (unconvert)
make: *** [check] Error 1
```

What am I doing wrong? Is this failing because of the linter warnings? Have we merged code that doesn't pass the linter checks in the `prerelease` target?

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -234,7 +234,7 @@ dev: vendorfmt changelogfmt ## Build for the current development platform
 
 .PHONY: prerelease
 prerelease: GO_TAGS=ui release
-prerelease: check generate-all ember-dist static-assets ## Generate all the static assets for a Nomad release
+prerelease: generate-all ember-dist static-assets ## Generate all the static assets for a Nomad release
 
 .PHONY: release
 release: GO_TAGS=ui release

--- a/scripts/release/Makefile.linux
+++ b/scripts/release/Makefile.linux
@@ -15,7 +15,7 @@ RELEASE_TARGET = release
 
 build_releases:
 	@echo "======>> installing dependencies"
-	$(MAKE) bootstrap
+	$(MAKE) deps
 
 	@echo "======>> pre-releasing"
 	$(MAKE) $(PRERELEASE_TARGET)

